etc/systemd/system/zfs-import.service.in

```diff
@@ -1,16 +1,15 @@
 [Unit]
-Description=Import ZFS pools by cache file
+Description=Import ZFS pool(s)
 Documentation=man:zpool(8)
 DefaultDependencies=no
 Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
 Before=dracut-mount.service
-ConditionPathExists=${ZPOOL_CACHE}
 
 [Service]
 Type=oneshot
 RemainAfterExit=yes
 EnvironmentFile=-@initconfdir@/zfs
 ExecStartPre=/sbin/modprobe zfs
-ExecStart=@sbindir@/zpool import ${ZPOOL_CACHE:+-c "${ZPOOL_CACHE}"} -aN ${ZPOOL_IMPORT_OPTS}
```

Comment:
I'm pretty sure we can write the one line equivalence of 

Pseudo code:

if cache exists; then
    use cache file
else if ZPOOL_IMPORT_PATH is set; then
    use ZPOOL_IMPORT_PATH
fi